Samples/WindowsAIFoundry/cs-winui/Util/SoftwareBitmapExtensions.cs

```diff
@@ -95,4 +96,71 @@ public static SoftwareBitmap ApplyMask(this SoftwareBitmap inputBitmap, Software
         segmentedBitmap.CopyFromBuffer(inputBuffer.AsBuffer());
         return segmentedBitmap;
     }
+
+    public static ImageBuffer ConvertToBgr8ImageBuffer(SoftwareBitmap input)
```

Comment:
Since the file name suggest 'Extension' i would mark these methods with 'this', it will be cleaner to read